GOVERNANCE-subprojects.md

```diff
@@ -18,6 +18,48 @@ The [TODO:PROJECTNAME] umbrella project is composed as a federation of individua
 some independent and some interdependent, each of which focuses on some aspect
 of [TODO:Goals of umbrella project].  Our governance reflects this federated structure.
 
+- [Values](#values)
+- [Individual Project Governance](#individual-project-governance)
+- [Steering Committee](#steering-committee)
+  - [Steering Committee Duties](#steering-committee-duties)
+  - [Steering Committee Elections](#steering-committee-elections)
+- [Code of Conduct Committee](#code-of-conduct-committee)
+- [Adding New Subprojects](#adding-new-subprojects)
+- [Removing Projects](#removing-projects)
+
+## Values
+
+<!-- This is where you put the core values or principles of your project, like
+openness, distributed design, fairness, diversity, etc.
+
+References and Examples
+* https://www.apache.org/theapacheway/
+* https://kubernetes.io/community/values/
+
+See https://contribute.cncf.io/maintainers/governance/charter for guidance and
+additional examples.  The values below are just example values as a jumping-off
+point for your project's actual values.  -->
+
+The [TODO: PROJECTNAME] and its leadership embrace the following values:
+
+* Openness: Communication happens in the open and is archived for future
+  reference. As much as possible, all discussions and work take place in public
+  forums and open repositories.
+
+* Fairness: All stakeholders have the opportunity to provide feedback and submit
+  contributions, which will be considered on their merits.
+
+* Community over Product or Company: Sustaining and growing our community takes
+  priority over shipping code or sponsors' organizational goals.  Each
+  contributor participates in the project as an individual.
+
+* Inclusivity: We innovate through different perspectives and skill sets, which
+  can only be heard in a welcoming and respectful environment.
```

Comment:
`which can only be heard` ... `heard` is this the right word?

Reply:
I'm changing this from "heard" to "accomplished", which matches the intent better.